L1Trigger/L1THGCal/src/be_algorithms/HGCalTriggerCellCalibration.cc

```diff
@@ -8,6 +8,10 @@ HGCalTriggerCellCalibration::HGCalTriggerCellCalibration(const edm::ParameterSet
     fCperMIP_fh_ = beCodecConfig.getParameter<std::vector<double>>("fCperMIPfh");
     dEdX_weights_ = beCodecConfig.getParameter<std::vector<double>>("dEdXweights");
     thickCorr_ = beCodecConfig.getParameter<std::vector<double>>("thickCorr");
+    
+    if(thickCorr_.at(0)==0 || thickCorr_.at(1)==0 || thickCorr_.at(2)==0){
```

Comment:
Better to have a loop instead of hardcoding the size.
Also I suppose that we can require `>0` (which is actually what you had just before)